### ISSUE TITLE
No database selected

### ISSUE THREAD

**Issue by tpitale** [opscode-cookbooks/database#149]
The database_name is set, and I've also added the database to connection info (based on the mysql 2 gem's docs). Still cannot run a query. It always responds 'No database selected'

```
    mysql_database("ias_bingo_production") do
      provider Chef::Provider::Database::Mysql
      action [:query]
      retries 0
      retry_delay 2
      default_guard_interpreter :default
      database_name "ias_bingo_production"
      declared_type :mysql_database
      cookbook_name :"ias-bingo-server"
      recipe_name "default"
      connection {:host=>"127.0.0.1", :port=>"3306", :username=>"<username>", :password=>"<password>", :database=>"ias_bingo_production"}
      sql "DROP TABLE IF EXISTS `daub_tweets`;\n\nCREATE TABLE …"
    end
    
[2015-04-11T11:47:54-04:00] ERROR: Converge failed with error message mysql_database[ias_bingo_production] (ias-bingo-server::default line 88) had an error: Mysql2::Error: No database selected
```

**Comment by eric-tucker** [comment]
This is an absolute blocker with no work-around.  It has been an issue for 5 months and there are two pull requests open to address the issue:

https://github.com/opscode-cookbooks/database/pull/129
https://github.com/opscode-cookbooks/database/pull/150

What is the hold up?  Is no one supporting this cookbook?  This is a CRITICAL BUG.

**Comment by pnguyen-okta** [comment]
This is blocking us also. Can we get this merged in? thanks.

**Comment by tas50** [comment]
This was merged in with #150
